README.md

```diff
@@ -55,20 +52,20 @@ Simply execute the following command in project root directory:
 ./gradlew clean build
 ```
 
-Make sure you have Java installed with JavaFX 8 at least. Liberica Full JDK 8 or later is recommended.
+Make sure you have Java installed with JavaFX 11 at least. Liberica Full JDK 11 or later is recommended.
```

Comment:
HMCL supports building with Java 8, and we recommend building HMCL with Java 8, because HMCL Jenkins is currently using Java 8.